modules/openxr/openxr_api.cpp

```diff
@@ -544,36 +547,64 @@ PackedStringArray OpenXRAPI::get_all_requested_extensions() {
 		}
 	}
 
+	// Also add in our OpenXR Version "extension", so we can switch logic on that.
+	requested_extensions.push_back(XR_OPENXR_1_1_NAME);
+
 	return requested_extensions;
 }
 
-bool OpenXRAPI::create_instance() {
-	// Create our OpenXR instance, this will query any registered extension wrappers for extensions we need to enable.
-
-	// We can request an extension multiple times if there are dependencies
-	struct RequestExtension {
-		String name;
-		bool *enabled;
-	};
+bool OpenXRAPI::is_promoted(XrVersion p_in_version, const char *p_extension) const {
+	// Check promotions in OpenXR 1.1.
+	if (p_in_version >= XR_MAKE_VERSION(1, 1, 0)) {
+		const char *promoted_extensions[] = {
+			XR_KHR_LOCATE_SPACES_EXTENSION_NAME,
+			XR_KHR_MAINTENANCE1_EXTENSION_NAME,
+			XR_EXT_HP_MIXED_REALITY_CONTROLLER_EXTENSION_NAME,
+			XR_EXT_LOCAL_FLOOR_EXTENSION_NAME,
+			XR_EXT_PALM_POSE_EXTENSION_NAME,
+			XR_EXT_SAMSUNG_ODYSSEY_CONTROLLER_EXTENSION_NAME,
+			XR_EXT_UUID_EXTENSION_NAME,
+			// XR_BD_CONTROLLER_INTERACTION_EXTENSION_NAME, // Note: we can't exclude this as pico4s_controller is not part of OpenXR 1.1
```

Comment:
Note to self, Bytedance is introducing a separate extension for the ultra controller. Will do a PR to support that and we can correctly filter out `XR_BD_CONTROLLER_INTERACTION_EXTENSION_NAME`